clang/lib/CodeGen/CGClass.cpp

```diff
@@ -2779,13 +2779,47 @@ void CodeGenFunction::EmitTypeMetadataCodeForVCall(const CXXRecordDecl *RD,
   }
 }
 
+void CodeGenFunction::ParseCFITypeCheckKind(CFITypeCheckKind TCK,
```

Comment:
return as a pair, try to avoid output arguments

Reply:
also just 2 function could OK as well

Reply:
Fixed in https://github.com/llvm/llvm-project/pull/139809/commits/fd3fd98add4cfadda22059dc6f175c08627ac282